src/factory_advanced.md

```diff
@@ -1,5 +1,11 @@
 # Advanced factories
 
+In this chapter we will build an even more advanced UI for modifying the available counters:
+
+![App screenshot dark](img/screenshots/factory-advanced-dark.png)
+
+Additionally one can now remove a certain counter or insert a counter between two existing ones.
```

Comment:
What do think about this: ...or insert a counter above or below an existing counter.